src/pages/home/HeaderView.js

```diff
@@ -24,11 +23,14 @@ import VideoChatButtonAndMenu from '../../components/VideoChatButtonAndMenu';
 import IOUBadge from '../../components/IOUBadge';
 import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
 import CONST from '../../CONST';
-import * as ReportUtils from '../../libs/reportUtils';
+import * as ReportUtils from '../../libs/ReportUtils';
 import Text from '../../components/Text';
 import Tooltip from '../../components/Tooltip';
 
 const propTypes = {
+    /** The ID of the report */
+    reportID: PropTypes.number.isRequired,
+
```

Comment:
Is it not available on the `report` anymore? Why was this change needed?

Reply:
Well, we were using the `reportID`, so documenting that prop makes sense. Removing it from the `report` wasn't necessary, but using `props.reportID` instead of `props.report.reportID` makes sense to me because they're the same thing in fewer characters.

It really doesn't matter though, so I can get rid of this change if you prefer.

Reply:
Will the `reportID` ever be different from the one on the `report` object? Maybe it makes sense to just use the `props.report.reportID` instead of have that and also pass `props.reportID`?

Reply:
Hmmm but isn't the `reportID` prop required to get a specific report from Onyx [here](https://github.com/Expensify/App/blob/main/src/pages/home/HeaderView.js#L205)?

Reply:
The `reportID` should always be provided [here](https://github.com/Expensify/App/blob/a1672dd1f513d3a80c83512bbc4146d604e5a7c5/src/pages/home/ReportScreen.js#L178), and the `report.reportID` will be there once Onyx loads. Since both values will be available, and they should always be the same, I sort of think it makes sense to use only the one that's explicitly passed in by the parent. But I don't think it really matters since they're the same, so I'm happy to change this to whatever you want.